public/lesswrong-editor/src/ckeditor5-footnote/theme/placeholder.css

```diff
@@ -0,0 +1,10 @@
+.placeholder {
+    padding: 2px 2px;
+    outline-offset: -2px;
+    line-height: 1em;
+    margin: 0 1px;
+}
+
+.placeholder::selection {
+    display: none;
+}
```

Comment:
[Files should end with newlines.](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)